### ISSUE TITLE
Copy/pasted text is behaving badly when text is imported from external file to the test documentation field

### ISSUE THREAD

**Issue by robotframework-ride** [robotframework/RIDE#1267]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1229) by miskammakela on 20 Mar 2013*

1) When I copy ascii text from unix type of file, line breaks are not handled properly in RIDE field.
2) Spaces are also converted weirdly with backslash+space combination when using similar copy/pasting. -&gt; Wrongly outputted text can be examined in "plain text tab".

**Comment by robotframework-ride** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1229#c2) by aki.siv... on 1 Apr 2014*


Noticed the same bug too.

1. Copy text from Notepad++ (ASCII)
2. Paste text to the RIDE's Documentation field 
--&gt; backslashes added.

**Comment by robotframework-ride** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1229#c3) by aki.siv... on 1 Apr 2014*


Version: RIDE v1.3
OS: Windows 7 64bit SP1
wxpython: 2.8.12.1

**Comment by yanne** [comment]
Yes, the documentation editor is trying to be to smart for it's own good.
